tests/unit/lms/validation/authentication/_lti_test.py

```diff
@@ -15,29 +15,15 @@
 
 class TestLTI11AuthSchema:
     def test_it_returns_the_lti_user_info(
-        self, schema, application_instance_service, lti_user_service
+        self, schema, application_instance_service, lti_user_service, pyramid_request
     ):
         lti_user = schema.lti_user()
 
-        lti_user_service.from_auth_params.assert_called_once_with(
+        lti_user_service.from_lti_params.assert_called_once_with(
             application_instance_service.get_by_consumer_key.return_value,
-            {
-                "oauth_signature_method": "SHA256",
-                "lis_person_name_family": "TEST_FAMILY_NAME",
-                "lis_person_contact_email_primary": "EMAIL",
-                "oauth_nonce": "TEST_NONCE",
-                "oauth_consumer_key": "TEST_OAUTH_CONSUMER_KEY",
-                "lis_person_name_full": "TEST_FULL_NAME",
-                "oauth_version": "1p0p0",
-                "lis_person_name_given": "TEST_GIVEN_NAME",
-                "oauth_signature": "TEST_OAUTH_SIGNATURE",
-                "roles": "Instructor",
-                "tool_consumer_instance_guid": "TEST_TOOL_CONSUMER_INSTANCE_GUID",
-                "user_id": "TEST_USER_ID",
-                "oauth_timestamp": "TEST_TIMESTAMP",
-            },
+            pyramid_request.lti_params,
         )
```

Comment:
:+1: Presumably there was something like this that could have been done before as well?